internal/pkg/service/buffer/api/service/service.go

```diff
@@ -34,18 +34,14 @@ import (
 type service struct {
 	deps   dependencies.ForServer
 	mapper mapper.Mapper
-	stats  statistics.Manager
+	stats  *statistics.APINode
 }
```

Comment:
Do balicka `statistics` pojde toho viac, ako je tam teraz.

Preto som premenoval `Manager -> APINode` + je to instancia, tak je lepsie pouzit pointer.
Zhodov okolnosti tam nemenis ziadny field, ktory sa neodovzdava cez refereciu, takze to ide aj bez `*`.
(`chan` a `map` su vzdy referencie, nikdy sa automaticky nekopiruju).

Podobny naming mame vo `worker`:
https://github.com/keboola/keboola-as-code/blob/2f045100cd42a84e5e2b49ec4ef327b6c8086ffb/internal/pkg/service/buffer/worker/service/service.go#L8-L11

Zaroven mi to viac evokuje, ze dana struktura zbera statistiky per node.